.drone.yml

```diff
@@ -1,4 +1,9 @@
 pipeline:
+  lint_kube_yaml:
+    image: fedora:25
+    commands:
+      - dnf install -q -y yamllint
+      - "yamllint -d '{extends: default, rules: {line-length: {max: 700}}}' kube/"
 
   my-build:
```

Comment:
Add `tag` and `deployment` to event

Reply:
It's not needed, I want this to run no matter what the event is.

Reply:
Then you should remove the `when` section that's already there